### DISCUSSION TITLE
Expanding selection up and on same row left/right

### DISCUSSION THREAD

**Discussion by imkitukov** [nick-keller/react-datasheet-grid#286] [Q&A] [answered]
Hey,

I saw when clicking on the bottom right corner of a cell you can drag and drop it into the same column and this way values on selected cells will be edited with the initially selected/dragged cell value. 

From examples, this looks supported only when the dragging direction is down on the same column of the initially selected cell/s/.

Is the same supported when dragging up or even left or right on the same row (I guess the columns may have to be of the same type of course)? 

![image](https://user-images.githubusercontent.com/44986252/230635220-caed7c36-c75a-438a-a39f-65f0176e2c21.png)

**Comment by nick-keller** [comment, marked as answer]
Hi,
Dragging is not supported horizontally because columns might be of different types. And it is not supported vertically for no particular reason, it should be pretty easy to implement if you give it a shot!
